### ISSUE TITLE
README.md needs to be more descriptive

### ISSUE THREAD

**Issue by wildsm** [bandframework/bandframework#16]
For a release, [README.md](https://github.com/bandframework/bandframework#readme) needs to contain information about what the software does, what it's aimed at, how to install and run, how to use, etc.
There are many guides online and examples. 

If the intention is that this information only be on the github.io website, then the specific location for this should be linked or reference (rather than the entire site).

**Comment by DanielRPhillips** [comment]
Happy to do this, but can only describe the software once I know what will be included in v1.

**Comment by wildsm** [comment]
Note that the readme in the develop branch in some respects defines what we might intend to have in next release. I will start by adding a table of contents in terms of what is currently there.

**Comment by DanielRPhillips** [comment]
I have updated README.md to try to provide some explanation of what the Framework is, what it seeks to accomplish, and what's in this release. This is in the develop branch right now.  @wildsm is this what you had in mind? If not, happy to add or subtract. 

I added a section "Downloading and using the BAND Framework" which encourages people to do those things. But that section could use an actual description of...what? How to clone the repo?
